rmf_site_editor/Cargo.toml

```diff
@@ -16,8 +16,8 @@ name = "extending_site_editor"
 path = "examples/extending_menu.rs"
 
 [dependencies]
-bevy_mod_outline = { git = "https://github.com/komadori/bevy_mod_outline", branch = "bevy-0.16"}
-bevy_gltf_export = { git = "https://github.com/xiyuoh/bevy_gltf_export", branch = "xiyu/bevy_0.16"}
+bevy_mod_outline = "0.10"
+bevy_gltf_export = { git = "https://github.com/luca-della-vedova/bevy_gltf_export" }
```

Comment:
Can you set a commit hash to the latest main here? To make sure it doesn't break unexpectedly

Reply:
[c73b477](https://github.com/open-rmf/rmf_site/pull/287/commits/c73b477a8f5f4c031dae0093d2f3779eff1f7db8)